Scoop/Controller/Home/NotificationsViewController.swift

```diff
@@ -51,18 +53,32 @@ class NotificationsViewController: UIViewController {
         tableView.contentInset = UIEdgeInsets(top: 20, left: 0, bottom: 20, right: 0)
         tableView.separatorColor = .systemGray
         tableView.separatorStyle = .singleLine
+        tableView.contentInsetAdjustmentBehavior = .never
         // Not sure about this, it makes the entire cell layout shift. Not sure how else to make the separator last the entire line though. 
 //        tableView.layoutMargins = UIEdgeInsets.zero
         tableView.register(RequestTableViewCell.self, forCellReuseIdentifier: requestCellIdentifier)
         view.addSubview(tableView)
         
         tableView.snp.makeConstraints { make in
-            make.top.equalToSuperview().inset(35)
+            make.top.equalToSuperview().inset(50)
             make.leading.trailing.equalToSuperview()
             make.bottom.equalTo(view.safeAreaLayoutGuide)
         }
     }
     
+    private func getRequests() {
+        NetworkManager.shared.getAllRequests { response in
```

Comment:
Use a weak instance of self here `arrivalID) { [weak self] in ` to avoid potential retain cycle